packages/vuetify/src/composables/proxiedModel.ts

```diff
@@ -46,6 +46,7 @@ export function useProxiedModel<
 
   const model = computed({
     get (): any {
+      void props[prop]
       return transformIn(isControlled.value ? props[prop] : internal.value)
```

Comment:
Save a proxy access?
```suggestion
      const externalValue = props[prop]
      return transformIn(isControlled.value ? externalValue : internal.value)
```